completions/bash/kpod

```diff
@@ -653,6 +653,66 @@ __kpod_images() {
 	__kpod_q images $images_args | awk "$awk_script" | grep -v '<none>$'
 }
 
+
+_kpod_build() {
+     local boolean_options="
+     --help
+     -h
+     --pull
+     --pull-always
+     --quiet
+     -q
+     --tls-verify
+  "
+
+     local options_with_args="
+     --signature-policy
+     --runtime
+     --runtime-flag
+     --tag
+     -t
+     --file
+     -f
+     --build-arg
+     --format
+  "
+
+     local all_options="$options_with_args $boolean_options"
+
+     case "$prev" in
+         --runtime)
+             COMPREPLY=($(compgen -W 'runc runv' -- "$cur"))
+             ;;
+         $(__buildah_to_extglob "$options_with_args"))
+ return
+ ;;
+ esac
+
+ case "$cur" in
+     -*)
+         COMPREPLY=($(compgen -W "$boolean_options $options_with_args" -- "$cur"))
+         ;;
+ esac
+}
+
+_kpod_diff() {
```

Comment:
Looks like a cut and paste error.

Reply:
Fixed.  Yeah looks like I double tapped ctrl-v.